daemon/api.go

```diff
@@ -721,16 +690,16 @@ func (inst *snapInstruction) dispatch() func() interface{} {
 	}
 }
 
-func pkgActionDispatchImpl(inst *snapInstruction) func() interface{} {
+func pkgActionDispatchImpl(inst *snapInstruction) func() (*state.Change, error) {
 	return inst.dispatch()
 }
 
 var pkgActionDispatch = pkgActionDispatchImpl
 
 func postSnap(c *Command, r *http.Request) Response {
-	route := c.d.router.Get(operationCmd.Path)
+	route := c.d.router.Get(stateChangeCmd.Path)
 	if route == nil {
-		return InternalError("router can't find route for operation")
+		return InternalError("router can't find route for change")
```

Comment:
Sorry, nevermind. There are tons of these in this file. I'll do a pass on it all later.